packages/omgidl-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@foxglove/omgidl-parser",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
We do this in a follow-up PR separate from the feature work.

Reply:
🤔 Interesting. Maybe in this repo that's been Sam's approach. For other repos (like I did ulog recently) we had separate PRs.

Reply:
Most of the time I've made separate PRs, but maybe not every time 😅